src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  for (let i = 1; i <= days; i++) {
+    result += 40;
+  }
+
+  if (days >= 7) {
+    result -= 50;
+  } else if (days >= 3) {
+    result -= 20;
```

Comment:
40, 50, and 20 still are magic numbers
try to keep all numbers like these in variables for more readable code